sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/models/Testing.java

```diff
@@ -0,0 +1,88 @@
+package com.azure.maps.search.models;
```

Comment:
Wrong package for this method. This class doesn't seem to have a reason for existing, needs to be removed (it's the same as SearchSample)